django_project_base/account/rest/impersonate.py

```diff
@@ -75,6 +74,21 @@ class ImpersonateUserDialogSerializer(serializers.Serializer):
 class ImpersonateUserViewset(viewsets.SingleRecordViewSet):
     serializer_class = ImpersonateUserDialogSerializer
 
+    def get_permissions(self):
```

Comment:
A je to cool? Dokumentacija pravi, da je treba to narest [tko](https://stackoverflow.com/a/72437418/1760858)... A to ni delalo?

Reply:
Kolkor vidim v dokumentaciji, dokumentacija prikazuje tako [moj](https://www.django-rest-framework.org/api-guide/viewsets/#introspecting-viewset-actions) in [predlagan](https://www.django-rest-framework.org/api-guide/viewsets/#marking-extra-actions-for-routing) nacin.

Karkoli je bilo pred tem za nastavljanje permissionov (per method) ni delovalo.